main_site.py

```diff
@@ -89,7 +90,9 @@ def render_team_page():
     """
     Renders the team page from jinja2 template
     """
-    return render_template("team.html")
+    with open('./team.json') as f:
```

Comment:
Should we do this once at app boot? Not familiar with Flask lifecycle.

Reply:
Since we're not concerned with performance, this is fine.